vuepress/docs/next/tutorials/getting-started/azure-install.md

```diff
@@ -80,11 +80,9 @@ ingress-nginx-controller  LoadBalancer  10.0.28.197   20.120.54.243
 ```
    Record the value of the EXTERNAL-IP.
 
-### Verify the NGINX Ingress Install
-We recommend verifying NGINX is working correctly. The simplest option is to access the EXTERNAL-IP in your browser. You should get a `404 Not Found` NGINX error page. Alternatively you can set up a simple test application. See [this page](../getting-started/gke-install#verify-the-nginx-ingress-install) for those steps. You can use either Azure Cloud Shell or your local `kubectl`.
-
-There are situations where the default NGINX ingress configuration doesn't work well for Entando and must be customized. Refer to the [Development Tips and Tricks](../../docs/reference/local-tips-and-tricks.md#customizing-nginx) page for more information.
-
+::: tip
+We recommend verifying NGINX is working correctly. This is confirmed with a `404 Not Found` error page when accessing the EXTERNAL-IP in your browser. Alternatively, you can [set up a simple test application](../devops/manage-nginx.md#verify-the-nginx-ingress-install) using either Azure Cloud Shell or your local `kubectl`. You can also [customize the NGINX ingress](../devops/manage-nginx.md#customize-the-ingress-configuration) to optimize the configuration for Entando.
+:::
 ### Install the Entando Custom Resource Definitions (CRDs)
 1. Download the Custom Resource Definitions (CRDs) and, once per cluster, deploy the cluster scoped resources
```

Comment:
use CRDs or Custom Resource Definitions in this sentence since the acronym is already defined above.